initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -937,6 +936,20 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="maxPersistentIdleTime" type="xs:integer" use="optional" default="60">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>The maximum idle time (-1 for unlimited, 0 to disable).</tooltip>
+                    </xs:appinfo>
+                </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="maxPersistentRequests" type="xs:integer" use="optional" default="100">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>The maximum number of persistent requests (-1 for unlimited, 0 to disable).</tooltip>
```

Comment:
Maximum number of query requests per persistent http connection. (-1 for unlimited, 0 to disable).